src/BuiltInTools/dotnet-watch/Browser/BrowserConnector.cs

```diff
@@ -117,6 +121,10 @@ public bool TryGetRefreshServer(ProjectGraphNode projectNode, [NotNullWhen(true)
 
             bool matchFound = false;
 
+            // Workaround for Aspire dashboard launching: scan for "Login to the dashboard at " prefix in the output and use the URL.
```

Comment:
Does this work in languages besides English?

Reply:
It won't. This is certainly not the ideal solution.

Reply:
Aspire does not localize the string right now: https://github.com/dotnet/aspire/blob/main/src/Shared/LoggingHelpers.cs